addons/sale_project/models/project.py

```diff
@@ -155,6 +155,7 @@ def action_open_project_invoices(self):
         query = self.env['account.move.line']._search([('move_id.move_type', '=', 'out_invoice')])
         query.add_where('analytic_distribution ? %s', [str(self.analytic_account_id.id)])
         query_string, query_param = query.select('DISTINCT move_id')
+        query_string = query_string.split('ORDER BY')[0]
```

Comment:
```suggestion
        query.order = None
        query_string, query_param = query.select('DISTINCT move_id')
```
this should be equivalent but a little more clean